example/lib/pages/home.dart

```diff
@@ -72,6 +75,15 @@ class Home extends StatelessWidget {
               },
               child: const Text('Casting'),
             ),
+            Visibility(
```

Comment:
I thought it would make sense to hide this example on Android for now since it's not yet supported there.